src/deprecated.jl

```diff
@@ -0,0 +1,10 @@
+using Base: @deprecate
+
+# BEGIN Intervals 0.1 deprecations
+
+@deprecate less_than_disjoint(a, b) is_less_than_disjoint(a, b) true
```

Comment:
Would it make sense to reuse some Base terminology here and call it like `isless_disjoint`?

Reply:
That is better. Do you think that `greater_than_disjoint(a, b)` should be deprecated to `isless_disjoint(b, a)`?

Reply:
Yeah, I think that seems reasonable.